language-server/src/main/java/org/sonarlint/languageserver/SonarLintLanguageServer.java

```diff
@@ -625,7 +599,7 @@ public AnalysisResultsWrapper analyze(Path baseDir, URI uri, String content, Iss
     @Override
     public boolean isExcludedByServerSideExclusions(URI fileUri) {
       Path baseDir = findBaseDir(fileUri);
-      ProjectBinding projectBinding = workspaceBindings.getOrDefault(fileUri.toString(), new ProjectBinding(projectKey, "", ""));
+      ProjectBinding projectBinding = workspaceBindings.getOrDefault(baseDir, new ProjectBinding(projectKey, "", ""));
```

Comment:
@jblievremont I think here I spotted a bug. The wrong map key was used, so always ending using the default value. It probably means issue exclusion was unreliable when path prefixes are not empty.